cmd/admin-handlers.go

```diff
@@ -72,7 +72,7 @@ type ServerStatus struct {
 // Fetches server status information like total disk space available
 // to use, online disks, offline disks and quorum threshold.
 func (adminAPI adminAPIHandlers) ServiceStatusHandler(w http.ResponseWriter, r *http.Request) {
-	adminAPIErr := checkRequestAuthType(r, "", "", "")
+	adminAPIErr := checkAdminRequestAuthType(r, "")
```

Comment:
If we are always passing the `region` argument as empty string, do we need to have it at all? @harshavardhana any idea why previous code also was doing this?

Reply:
If the region is set on server we need to enforce that, if not we allow all regions. This applies to Admin API as well . Need not be fixed in this PR.

Reply:
@donatello  Had same thought as you. Thanks @harshavardhana for details.